packages/big-design/src/components/Input/Input.tsx

```diff
@@ -10,7 +10,7 @@ import { StyledIconWrapper, StyledInput, StyledInputWrapper } from './styled';
 
 interface Props {
   description?: React.ReactChild;
-  error?: React.ReactChild;
+  error?: React.ReactChild | React.ReactChild[];
```

Comment:
How does this look when we pass an array of errors to an input not wrapped in a group?

Reply:
We removed the functionality in #129 in favor of always wrapping `Input`'s in `Form.Group`'s.

Reply:
Ahh true